hidapi/hidapi/dummy.go

```diff
@@ -0,0 +1,2 @@
+// go mod vendor prunes non-go dirs https://github.com/golang/go/issues/26366
+package hidapi_hidapi
```

Comment:
Add a blank line between the comment and the package line, because as is the comment is taken as package documentation.